src/openrct2/world/Scenery.h

```diff
@@ -116,7 +116,7 @@ enum WALL_SCENERY_FLAGS
     WALL_SCENERY_HAS_PRIMARY_COLOUR = (1 << 0),   // 0x1
     WALL_SCENERY_HAS_GLASS = (1 << 1),            // 0x2
     WALL_SCENERY_CANT_BUILD_ON_SLOPE = (1 << 2),  // 0x4
-    WALL_SCENERY_IS_BANNER = (1 << 3),            // 0x8  // Probably indicates translucency
+    WALL_SCENERY_IS_DOUBLE_SIDED = (1 << 3),            // 0x8  // Probably indicates translucency
```

Comment:
You'll need to align the comment for the ci to pass. I would remove the "probably..." comment it is no longer relevant.